Sport1Player/Sport1PlayerLivestreamAge.m

```diff
@@ -49,6 +75,13 @@ - (void)updateLivestreamAgeData {
     NSDictionary *current = [self currentLivestreamFromJSON:livestreamJSON];
     if (current) {
         self.livestreamEnd = [self dateFromString:current[kLivestreamEnd]];
+        if ([current.allKeys containsObject:kFSKKey]) {
+            self.fsk = current[kFSKKey];
+        } else {
+            self.fsk = nil;
+        }
+        
+        self.fsk = nil;
```

Comment:
This seems wrong

Reply:
Test line left in, but whole FSK checks have been removed.